ui/app/reducers/index.js

```diff
@@ -5,11 +5,17 @@ import clusters from 'reducers/ClusterReducer';
 import procs from 'reducers/ProcReducer';
 import profiles from 'reducers/ProfilesReducer';
 import aggregatedProfileData from 'reducers/AggregatedProfileDataReducer';
+import policyApps from 'reducers/PolicyAppReducer';
+import policyClusters from 'reducers/PolicyClusterReducer';
+import policyProcs from 'reducers/PolicyProcReducer';
 
 export default combineReducers({
   apps,
```

Comment:
profileApps and so on

Reply:
Renamed to profilesApps, similarly for clusters and procs